source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,362 @@
+
+After a migration is completed, Partition-Based Sync client apps will go through
+a client reset. Then, they can communicate with your App's Flexible Sync backend.
+You can disable and re-enable Flexible Sync and client apps will still
+communicate with your backend.
+
+If you re-enable Partitition-Based Sync you will lose the "migrated" status.
+This means you'll need to to another migration if you want to use Flexible Sync
+on the backend with Partition-Based Sync client apps.
+
+Migrate Permissions and Rules
+`````````````````````````````
+
+Your App's permissions and rules will be automatically migrated if the Partition-Based
+Sync rules have direct App Services Rules equivalents. This will override
+previously-defined App Services Rules.
+
+In step 4 above, you will see if your permissions and rules can be automatically
+migrated. On the :guilabel:`Review Migration Process` step in the App Services UI,
+look for :guilabel:`Rules & Permisssions`.
+
+Some Partition-Based Sync rule strategies can't translate directly to App Services
+Rules. You may need to manually migrate permissions that include:
+
+- :json-operator:`%function` operator.
+
+  :ref:`Function rules <pbs-function-rules>` are not compatible with Flexible Sync 
+  and cannot be migrated.
+
+- ``%or``, ``%not``, ``%nor``, ``%and`` expansions.
+
+  These permissions may work, but there is enough nuance that you should test them
+  to ensure expected behavior. Testing new permissions won't work on the App
+  you are migrating. Instead, create a new Flexible Sync app to test your
+  manually-migrated permissions. Then apply the tested permissions to the app
+  you are migrating.
+
+See the list of :ref:`Flexible Sync-compatible expansions <sync-compatible-expansions>`
+for all supported expansions.
+
+You should also check out the 
+:ref:`Device Sync Permissions Guide <flexible-sync-permissions-guide>` for more
+information about how to work with permissions.
+
+.. _realm-sync-migrate-cancel:
+
+Cancel Migration
+~~~~~~~~~~~~~~~~
+
+You can cancel a migration at any time while it is in progress. This will return
+your App the Partition-Based Sync, with all settings as they were before the
+migration started.
+
+If you cancel a migration, you need to start over from the beginning with future
+migration attempts.
+
+**To cancel a migration:** 
+
+#. In the App Services UI, go to your App and select the Device Sync page.
+#. At the top of the page, in the notification banner, select
+   :guilabel:`Stop Migration`.
+   
+   .. image:: /images/cancel-migration.png
+      :alt: Cancel a migration
+   
+   This will cancel your migration. You will need to migrate again if you want to
+   enable Flexible Sync in the future.
+
+.. _realm-sync-migrate-commit:
+
+Commit Migration
+~~~~~~~~~~~~~~~~
+
+After your migration has completed, it is in an evaluation state. You can manually
+commit the migration to make it permanent or revert the migration to return your
+App to Partiton-Based Sync.
```

Comment:
partition**

Reply:
Nice catch! Thanks for pointing this out.